Katydid/Regex/Calculus.lean

```diff
@@ -129,17 +136,17 @@ def derive_scalar {α: Type} {a: α} {s: Prop} {P: Lang α}:
   rfl
 
 def derive_concat {α: Type} {a: α} {P Q: Lang α} {w: List α}:
-  (derive (concat P Q) a) w <-> (scalar (null P) (or (derive Q a) (concat (derive P a) Q))) w := by
+  (derive (concat P Q) a) w <-> (or (concat (derive P a) Q) (scalar (null P) (derive Q a))) w := by
```

Comment:
scalar is used as a short cut for an if statement, so I think was incorrectly translated before